docs/index.bs

```diff
@@ -1576,11 +1576,16 @@ spec: storage; urlPrefix: https://storage.spec.whatwg.org/
       The <dfn method for="InstallEvent"><code>addRoutes(|rules|)</code></dfn> method steps are:
 
         1. Let |routerRules| be a list of {{RouterRule}} dictionaries.
+        1. Let |hasFetch| be a boolean, initially false.
```

Comment:
Probably `hasFetchEventSource` is clear.

Reply:
Renamed